api/tests/utils/crypto_test.py

```diff
@@ -17,6 +19,14 @@ def test_check_public_api_key(self):
         assert not crypto.check_public_api_key("wrong", hashed)
         assert crypto.check_public_api_key("secret", hashed)
 
+    def test_no_password_leak(self):
+        non_unicode_password = "user@AZERTY123\udee0"
+
+        with pytest.raises(Exception) as exception:
+            crypto.hash_password(non_unicode_password)
+
+        assert "user@AZERTY123" not in str(exception), str(exception)
```

Comment:
Je ne comprends pas pourquoi `, str(exception)`, subtilité ou oubli ?

```suggestion
        assert "user@AZERTY123" not in str(exception)
```

Reply:
Le deuxième argument de `assert` est affiché quand le premier argument n'est pas truthy

Je n'ai pas réussi à faire passer le test du premier coup, donc j'ai affiché l'exception pour m'aider à débugger. Je ne vois pas de raison d'enlever le deuxième argument maintenant qu'il est là

Reply:
ok ça me va :)